sdk/identity/azure-identity/src/main/java/com/azure/identity/package-info.java

```diff
@@ -2,7 +2,218 @@
+ * served by the default settings should use other credential types.
+ * For more information refer to <a href="https://aka.ms/azsdk/java/identity/defaultazurecredential/docs"> Conceptual knowledge and configuration details </a>.
+ *
+ * <p><strong>Sample: Construct a simple DefaultAzureCredential</strong></p>
+ * <!-- src_embed com.azure.identity.credential.defaultazurecredential.construct -->
+ * <pre>
+ * DefaultAzureCredential defaultAzureCredential = new DefaultAzureCredentialBuilder&#40;&#41;
+ *     .build&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.identity.credential.defaultazurecredential.construct -->
+ *
+ *
+ * <H2> Authenticating on Azure Hosted Platforms via Managed Identity</H2>
+ * The Managed Identity authenticates the managed identity (system or user assigned) of an Azure resource. So, if the
+ * application is running inside an Azure resource that supports Managed Identity through IDENTITY/MSI, IMDS endpoints,
+ * or both, then the ManagedIdentityCredential will get your application authenticated, and offers a great secretless authentication experience.
+ * For more information refer to <a href="https://aka.ms/azsdk/java/identity/managedidentity/docs"> Conceptual knowledge and configuration details </a>.
+ *
+ * <p><strong>Sample: Construct a Managed Identity Credential</strong></p>
+ * <!-- src_embed com.azure.identity.credential.managedidentitycredential.construct -->
+ * <pre>
+ * ManagedIdentityCredential managedIdentityCredential = new ManagedIdentityCredentialBuilder&#40;&#41;
+ *     .clientId&#40;clientId&#41; &#47;&#47; specify client id only if targeting a user-assigned managed identity.
+ *     .build&#40;&#41;;
+ * </pre>
+ * <!-- end com.azure.identity.credential.managedidentitycredential.construct -->
+ *
+ * For other credentials that work well in Azure Hosted platforms, refer to the table below.
+ * <table style="border: 2px; width: 50%;">
+ *   <caption><strong>Authenticate Azure-hosted applications</strong></caption>
+ *   <thead>
+ *     <tr>
+ *       <th>Credential class</th>
+ *       <th>Usage</th>
+ *     </tr>
+ *   </thead>
+ *   <tbody>
+ *     <tr>
+ *       <td>{@link com.azure.identity.DefaultAzureCredential}</td>
+ *       <td>Provides a simplified authentication experience to quickly start developing applications run in Azure</td>
+ *     </tr>
+ *     <tr>
+ *       <td>{@link com.azure.identity.ChainedTokenCredential}</td>
+ *       <td>Allows users to define custom authentication flows composing multiple credentials</td>
+ *     </tr>
+ *     <tr>
+ *       <td>{@link com.azure.identity.EnvironmentCredential}</td>
+ *       <td>Authenticates a service principal or user via credential information specified in environment variables</td>
+ *       <td></td>
+ *     </tr>
+ *     <tr>
+ *       <td>{@link com.azure.identity.ManagedIdentityCredential}</td>
+ *       <td>Authenticates the managed identity of an Azure resource</td>
+ *     </tr>
+ *   </tbody>
+ * </table>
+ *
+ *
```

Comment:
"Refer to the JavaDoc for each of these classes for more details about when these credential types should be used."